cmd/skopeo/proxy.go

```diff
@@ -619,9 +620,8 @@ func (h *proxyHandler) GetBlob(args []any) (replyBuf, error) {
 
 // GetLayerInfo returns data about the layers of an image, useful for reading the layer contents.
 //
-// This needs to be called since the data returned by GetManifest() does not allow to correctly
-// calling GetBlob() for the containers-storage: transport (which doesn’t store the original compressed
-// representations referenced in the manifest).
+// This is the same as GetLayerInfoPiped, but returns its contents inline. This is subject to
+// failure for large images and is only retained for backwards compatibility.
```

Comment:
I don’t think there is a Skopeo-side failure – “hard to handle in callers which want to impose a limit on the total size of the initial `reply`” maybe?

Reply:
I clarified this in the comment